test/end-to-end/company.js

```diff
@@ -170,7 +170,7 @@ describe("Company application endpoint", () => {
 
                     expect(res.body.errors).toContainEqual({
                         "location": "body",
-                        "msg": "file-too-large",
+                        "msg": "file-cant-be-larger-than-10mb",
```

Comment:
Import the constant from `utils` and use it here, the test should not fail if we simply change the message constant